core/src/main/java/cz/xtf/core/openshift/HelmBinary.java

```diff
@@ -0,0 +1,56 @@
+package cz.xtf.core.openshift;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import cz.xtf.core.config.OpenShiftConfig;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class HelmBinary {
+
+    private final String path;
+    private final String helmConfigPath;
+    private final String kubeUsername;
+    private final String kubeToken;
+    private final String namespace;
+
+    public HelmBinary(String path, String kubeUsername, String kubeToken, String namespace) {
+        this.path = path;
+        this.kubeUsername = kubeUsername;
+        this.kubeToken = kubeToken;
+        Path helmConfigFile = Paths.get(path).getParent().resolve(".config");
+        try {
+            helmConfigFile = Files.createDirectories(helmConfigFile);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        this.helmConfigPath = helmConfigFile.toAbsolutePath().toString();
+        this.namespace = namespace;
+    }
+
+    public HelmBinary(String path, String helmConfigPath, String kubeUsername, String kubeToken, String namespace) {
+        this.path = path;
+        this.helmConfigPath = helmConfigPath;
+        this.kubeUsername = kubeUsername;
+        this.kubeToken = kubeToken;
+        this.namespace = namespace;
+    }
+
+    public String execute(String... args) {
+        Map<String, String> environmentVariables = new HashMap<>();
+        environmentVariables.put("HELM_CONFIG_HOME", helmConfigPath);
```

Comment:
My feeling is we should use directly command parameters if this is possible instead of env variables.

Environmnet variables can be used to tweak behaviour from outside. but I do not have strong opinion.

Reply:
Ok